nav2_bringup/README.md

```diff
@@ -17,32 +17,37 @@ Pre-requisites:
 * A Gazebo world for simulating the robot (see Gazebo tutorials)
 * A map of that world saved to a map.pgm and map.yaml (see ROS Navigation tutorials)
 
-Launch Gazebo and Rviz2
+Terminal 1: Launch Gazebo and Rviz2
 
-`ros2 launch nav2_bringup gazebo_rviz2_launch.py world:=<full/path/to/gazebo.world>`
+```
+export GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:<full/path/to/turtlebot3_simulations/turtlebot3_gazebo/models>
```

Comment:
We should either change this to a Turtlebot3 based tutorial or keep this more generic and like </full/path/to/my_robot/gazebo/models>

Reply:
You could also have an example like in the Terminal 2 section below.

Reply:
fixed in 9fa61d3